specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -648,8 +655,8 @@
               "$ref": "#/definitions/ConstrainedResource"
             }
           },
-          "200": {
-            "description": "Success",
+          "202": {
```

Comment:
We recommend to follow 200 + ProvisioningState pattern. Please see here: https://github.com/Azure/azure-resource-manager-rpc/blob/e64480d4f830bd3ac18286e4ee1b2769ca7283d4/v1.0/Addendum.md#creatingupdating-using-put

Reply:
The update operation is async as well, this pattern mean returning Azure-AsyncOperation as part of a 200 OK result ?

Reply:
(Also see comment above about API alignment)